.github/workflows/CD.yml

```diff
@@ -320,27 +321,32 @@ jobs:
           key: "${{ env.PACKAGE_CACHE_KEY }}"
           path: build/packages
 
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
+      - name: Login to Public Release ECR
+        id: login-ecr
+        uses: docker/login-action@v1
         with:
-          aws-access-key-id: ${{ secrets.RELEASE_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.RELEASE_SECRET }}
-          aws-region: us-west-2    
-          
+          registry: public.ecr.aws
+          username: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
+          password: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
+        env:
+          AWS_REGION: us-east-1
+
       - name: Install AWS Cli 2.0
         run: |
           curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
           unzip awscliv2.zip
           sudo ./aws/install --update
 
-      - name: Load Image
-        run: docker load < build/packages/$IMAGE_NAME.tar
-        
-      - name: upload to ECR
+      - name: Pull image from integration test ECR
```

Comment:
Can use https://github.com/akhilerm/tag-push-action to pull/retag/push. I think you should be able to login to both of the public ECRs before using the action.

Reply:
Fixed it. Currently using akhilerm/tag-push-action. An example would be [this](https://github.com/aws-observability/aws-otel-collector/pull/860/files#diff-a64cbb6e953e697cb4302ab31af9ebbd95a8de6216f874bb0ffdefce8e3c1cd9R525-R539)